extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -482,6 +501,20 @@ export class AmpSlideScroll extends BaseSlides {
   }
 
   /**
+   * Shows the slide at the given index and triggers a `slidechange` action.
+   * @param {number} newIndex
+   * @private
+   */
+  showSlideAndTriggerAction_(newIndex) {
```

Comment:
I think we still need it since it's called from `updateOnScroll_` and scroll events are not supported by the base class.